pkg/k8shandler/status.go

```diff
@@ -199,9 +199,9 @@ func isPodReady(pod v1.Pod) bool {
 	return true
 }
 
-func (er *ElasticsearchRequest) updateNodeConditions(status *api.ElasticsearchStatus) {
-	esClient := er.esClient
-	cluster := er.cluster
+func (elasticsearchRequest *ElasticsearchRequest) updateNodeConditions(status *api.ElasticsearchStatus) {
```

Comment:
These were lint failures that I fixed. Although I don't believe these should be long names according to the official go documentation, they should definitely all match.

Reply:
we have a card to unify all the names to be shorter like `er` 
https://issues.redhat.com/browse/LOG-824

Reply:
I'm 100% on board with `er`